node/src/tasks/bsp_upload_file.rs

```diff
@@ -53,6 +56,7 @@ where
 {
     storage_hub_handler: StorageHubHandler<FL, FSH>,
     file_key_cleanup: Option<H256>,
+    capacity_queue: Arc<Mutex<Vec<u64>>>,
```

Comment:
I might be missing something, but as far as I understand this could be just a `u64` instead of `Vec<u64>` - seems this is the way we are using it.

Reply:
We could have a unique value that keep track of the total capacity we need to cover. I believe both ways are valid and with little impact.